libkbfs/impatient_debug_dumper.go

```diff
@@ -119,7 +119,20 @@ func NewImpatientDebugDumper(config Config, dumpIn time.Duration) *ImpatientDebu
 	return d
 }
 
-func (d *ImpatientDebugDumper) dump(tracker *ctxTimeTracker) {
+// NewImpatientDebugDumperForForcedDumps creates a new
+// *ImpatientDebugDumper, just as above, though without launching any
+// background goroutines or allowing the user to begin any
+// time-tracked tasks
```

Comment:
Missing period. Also, with this dumper, is calling `ForceDump` the only way to trigger a dump? If so would you mind adding that to comment?